egs/wsj/s5/steps/nnet3/train_raw_dnn.py

```diff
@@ -360,7 +365,7 @@ def learning_rate(iter, current_num_jobs, num_archives_processed):
 
         num_archives_processed = num_archives_processed + current_num_jobs
 
-    if args.stage <= num_iters:
+    if args.stage <= num_iters and args.final_combination:
```

Comment:
we'l want to have it force-symlink (function force_symlink in libs/common.py) the last model to final.mdl, if final_combination was false, so this would work if the next stage was ASR decoding.  [this would require an if-statement inside the existing if-statement].